src/system/mod.rs

```diff
@@ -600,7 +600,7 @@ impl Process {
     /// attached to the given process
     pub fn tty_device_id(pid: WithProcess) -> std::io::Result<Option<DeviceId>> {
         // device id of tty is displayed as a signed integer of 32 bits
-        let data: i32 = read_proc_stat(pid, 6)?;
```

Comment:
shouldn't we either
- in the comments note what 'field 6/7' is
- make these a `const` with a descriptive name.

Note: if original sudo has a similar routine (I don't remember if it has or we factored `read_proc_stat` out ourselves), we should probably use the same "0-based" or "1-based" convention as that.

Reply:
Original sudo doesn't have it factored out: https://github.com/sudo-project/sudo/blob/17aa7688c955e58adffdfb0300d485a2b859b128/plugins/sudoers/starttime.c#L196-L266 It does use 1-based for the actual field comparison though, which is why I made this change in the first place: https://github.com/sudo-project/sudo/blob/17aa7688c955e58adffdfb0300d485a2b859b128/plugins/sudoers/starttime.c#L228

Reply:
Added block comments with the official name of each field.